### PR TITLE
redirect and profile pull down menu 

### PR DESCRIPTION
**Pull request by FlyingUnicornSF**
redirect and profile pull down menu  still not functioning correctly.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6838 @@
+{
```
Comment (jayrbolton) [inline package-lock.json]: Remove this file plz. We are just using yarn

File: src/client/index.tsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ if (CONTAINER) {
 
   if (PATHNAME in routes) {
     const topComponent = routes[PATHNAME].component;
-    render(<Page root={topComponent} />, CONTAINER);
+    ReactDOM.render(<Page root={topComponent} />, CONTAINER);
```
Comment (jayrbolton) [inline src/client/index.tsx]: Aren't we importing `render` directly on line 2? Why this change?

File: src/client/js/components/dashboard/NarrativeList/ItemList.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
+import React, {Component} from 'react';
```
Comment (jayrbolton) [inline src/client/js/components/dashboard/NarrativeList/ItemList.js]: Junk file here to remove

**Comment by jayrbolton** [comment]
I did a partial review, but I don't see the changes you're talking about in the diff. Maybe you want to merge a different branch from your fork?

**Comment by jayrbolton** [comment]
@FlyingUnicornSF It looks like you already pushed your changes directly to master...

**Comment by jayrbolton** [comment]
Maybe a little reading on git would be good? https://rogerdudler.github.io/git-guide/

**Comment by FlyingUnicornSF** [comment]
> Maybe a little reading on git would be good? https://rogerdudler.github.io/git-guide/

thank you for the link but this one doesn't explain different workflow with a forked repo.  
when you fork a repo and make a "Feature Branch", you can't push the "Feature Branch" back into the origin repo, unless the origin has the "Feature Branch".   If I want to pull my local "Feature Branch" to the origin then I have to merge it to the master locally and PR to the origin master.  If there is a better way to maintain "feature branch" then let me know.

**Comment by jayrbolton** [comment]
Oh I thought it did but you're right. Maybe this one?
https://gist.github.com/Chaser324/ce0505fbed06b947d962

**Comment by jayrbolton** [comment]
Making a feature branch on your fork and then making a PR from this repo to your fork (the branch doesn't matter) is the way to go. Before you opened this PR, you pushed the commits directly to the master branch of this repo.

**Comment by FlyingUnicornSF** [comment]
that link helps!! 
so rebase master/origin, then PR to from forked/feature-branch -> origin/master to avoid merge conflict?

**Comment by jayrbolton** [comment]
I personally wouldn't rebase. If there are new upstream changes on master,
then just merge master into your branch.
